### ISSUE TITLE
color of Why Rust? section not accessible

### ISSUE THREAD

**Issue by tspiteri** [rust-lang/beta.rust-lang.org#420]
White text on the green background in the *Why Rust?* section does not have enough contrast. A foreground of `#FFFFFF` on a background `#46A092` has a contrast ratio of 3.13 which is below the 4.5 minimum required for AA compliance, that is it is not accessible (see http://accessible-colors.com).

**Comment by ashleygwilliams** [comment]
thank you for filing @tspiteri - i was using a tool (google lighthouse) and had bumped the contrast, but it appears the tool i used didn't have the right thresholds- i would accept a PR that bumped the green to a similar but more accessible color!
